src/repository/MongoDb.js

```diff
@@ -53,7 +53,8 @@ class MongoDb extends BaseMongoDb implements IBaseDatabase {
         if (skip) {
           result = result.skip(parseInt(skip, 10));
         }
-        if (take) {
+
+        if (take && parseInt(take, 10) !== 0) {
```

Comment:
We should have better checks here for both `skip` and `take`

`null`/`undefined` should mean fetch all. Passing in `0` should be valid and get passed into the query.

Reply:
ok, I have to remove default take value then.